cmd/banner.go

```diff
@@ -42,7 +42,7 @@ func supervisorGet(section string, command string) (outdata *(map[string]any), e
 			outdata = &(data.Data)
 		}
 	} else {
-		return nil, fmt.Errorf("error returned from Supervisor: %s", data.Message)
+		return nil, fmt.Errorf("Error returned from Supervisor: %s", data.Message)
```

Comment:
Linter says:

```
  cmd/banner.go:45:15: ST1005: error strings should not be capitalized (staticcheck)
```

So

```suggestion
		return nil, fmt.Errorf("error returned from Supervisor: %s", data.Message)
```